### PR TITLE
Feature/smal improvements

### LINKED ISSUE #207: Improvements
### Please describe the feature you would like to see implemented.

- [x] Hide Performance-Table if there is no recommendation
- [x] Add logger when restoring changed Shopware Files
- [x] Add note about config `frosh_tools.file_checker.exclude_files`

### PR DESCRIPTION
**Pull request by tinect**
Closes #207

### CODE REVIEW

File: src/Controller/ShopwareFilesController.php [hunk 1]
```diff
@@ -143,7 +151,13 @@ public function restoreShopwareFile(Request $request): JsonResponse
             opcache_reset();
         }
 
-        return new JsonResponse(['status' => sprintf('File at "%s" has been restored', $file)]);
+        $userName = $this->getUserName($context) ?? 'unknown';
+
+        $message = sprintf('File at "%s" has been restored by %s', $file, $userName);
```
Comment (shyim) [inline src/Controller/ShopwareFilesController.php:156]: you can use parameter {name} and replace them with context see https://symfony.com/doc/current/logging.html

File: src/Controller/ShopwareFilesController.php [hunk 2]
```diff
@@ -175,4 +189,22 @@ private function isIgnoredFileHash(string $file): int
...
+    private function getUserName(Context $context): ?string
+    {
+        $contextSource = $context->getSource();
+
+        if (!$contextSource instanceof AdminApiSource) {
+            return null;
+        }
+
+        /** @var null|UserEntity $userEntity */
+        $userEntity = $this->userRepository->search(new Criteria([$contextSource->getUserId()]), Context::createDefaultContext())->first();
```
Comment (shyim) [inline src/Controller/ShopwareFilesController.php:202]: getUserId can be null when integrationId is filled

File: src/Controller/ShopwareFilesController.php (hunk 1 above)
Comment (tinect) [reply src/Controller/ShopwareFilesController.php:156]: Uhh, nice to know. But we use it for the result, too.

File: src/Controller/ShopwareFilesController.php (hunk 2 above)
Comment (tinect) [reply src/Controller/ShopwareFilesController.php]: Thank you! Integration is now supported, too.

**Review by shyim — APPROVED** [review]

**Pull request merged**